spdx/parsers/tagvaluebuilders.py

```diff
@@ -724,8 +724,10 @@ def set_pkg_files_analyzed(self, doc, files_analyzed):
             if files_analyzed:
                 if validations.validate_pkg_files_analyzed(files_analyzed):
                     self.package_files_analyzed_set = True
-                    doc.package.files_analyzed = files_analyzed
-                    print(doc.package.files_analyzed)
+                    doc.package.files_analyzed = (files_analyzed.lower() == 'true')
+                    # convert to boolean;
```

Comment:
May be you meant to remove these comments?

Reply:
It was just a reminder :)

Please remove them

Reply:
```suggestion
                    doc.package.files_analyzed = (files_analyzed.lower() == 'true')
```

Reply:
```suggestion
```